trillian/ctfe/ct_server/main.go

```diff
@@ -169,10 +169,10 @@ func main() {
 		}
 		res.InitialState(resolver.State{Addresses: addrs})
 		resolver.SetDefaultScheme(res.Scheme())
-		dialOpts = append(dialOpts, grpc.WithBalancerName(roundrobin.Name), grpc.WithResolvers(res))
+		dialOpts = append(dialOpts, grpc.WithDefaultServiceConfig(fmt.Sprintf(`{"loadBalancingConfig": [{"%s":{}}]}`, roundrobin.Name)), grpc.WithResolvers(res))
```

Comment:
I appreciate the effort to keep some usage of the constants here, but given that this is a) difficult to read, and b) already contains a "magic string" for `loadBalancingConfig` key anyway, I'd be tempted to do what https://github.com/grpc/grpc-go/blob/master/examples/features/load_balancing/client/main.go#L77 does, here and below:

```suggestion
		dialOpts = append(dialOpts, grpc.WithDefaultServiceConfig(`{"loadBalancingConfig": [{"round_robin":{}}]}`), grpc.WithResolvers(res))
```

Is it just me, or is configuring options using json instead of a struct a bit weird? Anyway, thanks for doing this!

Reply:
I agree that it's probably a better idea to inline this.

Reply:
thanks so much the feedback and review